tests/unit/test_migration.py

```diff
@@ -368,7 +368,7 @@ def test_sequence_number_update_after_migration(app, location, script_info):
 
     # run seq number update
     runner = CliRunner()
-    res = runner.invoke(cli_sequence_generator, ['2017'], obj=script_info)
+    res = runner.invoke(cli_sequence_generator, ['2018'], obj=script_info)
```

Comment:
Speaking out of intuition (and with no much idea as well): Will this fail next year? if so...maybe we should `get_current_year` :)